index.html

```diff
@@ -226,6 +226,17 @@ <h2>Introduction</h2>
 [[NIST-SP-800-186]]. [[FIPS-186-5]] includes the <em>deterministic</em> ECDSA
 algorithm which is also specified in [[RFC6979]].
       </p>
+
+      <p class="note">
+The elliptic curves P-256 and P-384 of [[NIST-SP-800-186]] are referred to as
+<em>secp256r1</em> and <em>secp384r1</em> respectively in [[SECG2]]. This notation 
+is sometimes used in ECDSA software libraries. Developers 
+must be careful not to confuse <em>secp256<b>r</b>1</em> terms with 
+<em>secp256<b>k</b>1</em>: the second is a <em>different</em> elliptic curve that is not
+used by this specification. ECDSA software libraries may implement only one of the two 
+alternatives.
```

Comment:
This would be better with a link/citation to the relevant ECDSA or other RFC/TR/whatever.
```suggestion
used by this specification. ECDSA software libraries only need to implement one of the
two alternatives to be standards compliant.
```

Reply:
I am not sure about this. I do not think there is such a "standard compliant" software definition. The two curves are simply different mathematically, and software might decide to do only NIST compliance (and go with the "r" version) or bitcoin compliant (and go with the "k" version). From a distance, it looks like competing "standards" coming from different groups. (@msporny, is that correct?)

I'd prefer to stay away from this mess here; the original formulation is neutral. Unless it is grammatically problematic, I would prefer to stay with it.

Reply:
Given [this comment](https://github.com/w3c/vc-di-ecdsa/pull/68/files#r1714737595), and several re-readings of this paragraph, I have a [larger change suggestion](https://github.com/w3c/vc-di-ecdsa/pull/68/files#r1715747802) which also removes the lowercase RFC2119 terms.

Reply:
Marking this as resolved, because the larger change suggestion has been accepted.